tests/test_fremdkostenposition.py

```diff
@@ -0,0 +1,115 @@
+                    "zeitmenge": None,
+                    "gebietcodeEic": None,
+                    "betragKostenposition": {"wert": "12.5", "waehrung": "EUR"},
+                    "linkPreisblatt": None,
+                },
+            ),
+            pytest.param(
+                Fremdkostenposition(
+                    positionstitel="Vadders Preisstaffel",
+                    von=datetime(2013, 5, 1, tzinfo=timezone.utc),
+                    bis=datetime(2014, 5, 1, tzinfo=timezone.utc),
+                    artikelbezeichnung="Deim Vadder sei Preisstaffel",
+                    link_preisblatt="http://foo.bar/",
+                    zeitmenge=example_menge,
+                    einzelpreis=Preis(
+                        wert=Decimal(3.50),
+                        einheit=Waehrungseinheit.EUR,
+                        bezugswert=Mengeneinheit.KWH,
+                        status=Preisstatus.ENDGUELTIG,
+                    ),
+                    betrag_kostenposition=Betrag(
+                        waehrung=Waehrungscode.EUR,
+                        wert=Decimal(12.5),
+                    ),
+                    menge=example_menge,
+                    artikeldetail="foo",
+                    marktpartnercode="986543210123",
+                    marktpartnername="Mein MP",
+                    gebietcode_eic="not an eic code but validation will follow in ticket 146",
+                ),
+                {
+                    "marktpartnername": "Mein MP",
+                    "artikelbezeichnung": "Deim Vadder sei Preisstaffel",
+                    "menge": {"einheit": "MWH", "wert": "3.410000000000000142108547152020037174224853515625"},
+                    "artikeldetail": "foo",
+                    "zeitmenge": {"einheit": "MWH", "wert": "3.410000000000000142108547152020037174224853515625"},
+                    "positionstitel": "Vadders Preisstaffel",
+                    "marktpartnercode": "986543210123",
+                    "bis": "2014-05-01T00:00:00+00:00",
+                    "von": "2013-05-01T00:00:00+00:00",
+                    "einzelpreis": {"einheit": "EUR", "status": "ENDGUELTIG", "wert": "3.5", "bezugswert": "KWH"},
+                    "betragKostenposition": {"wert": "12.5", "waehrung": "EUR"},
+                    "gebietcodeEic": "not an eic code but validation will follow in ticket 146",
+                    "linkPreisblatt": "http://foo.bar/",
+                },
+            ),
+        ],
+    )
+    def test_serialization_roundtrip(self, fremdkostenposition: Preisstaffel, expected_json_dict: dict):
+        """
+        Test de-/serialisation of Preisstaffel.
+        """
+        assert_serialization_roundtrip(fremdkostenposition, FremdkostenpositionSchema(), expected_json_dict)
+
+    def test_missing_required_attribute(self):
+        with pytest.raises(TypeError) as excinfo:
+            _ = Fremdkostenposition()
+
+        assert "missing 13 required" in str(excinfo.value)
```

Comment:
Sind nur 4 required oder?
```suggestion
        assert "missing 4 required" in str(excinfo.value)
```

Reply:
e71a46b93d0216a5a6fb5e9eb930bac1ee8697c5